### PR TITLE
Updated the umberto config

### PR DESCRIPTION
**Pull request by przemyslaw-zan**
### Suggested merge commit message ([convention](https://ckeditor.com/docs/ckeditor5/latest/framework/contributing/git-commit-message-convention.html))

Internal: Added domain of nightly docs to `permalinkDisableDomains` in umberto config.

---

### Additional information

_For example – encountered issues, assumptions you had to make, other affected tickets, etc._

### CODE REVIEW

File: docs/umberto.json [hunk 1]
```diff
@@ -13,6 +13,9 @@
 			"**/index.html"
 		]
 	},
+	"permalinkDisableDomains": [
+		"ckeditor5.github.io"
```
Comment (pomek) [inline docs/umberto.json:17]:
```suggestion
		"ckeditor5.github.io",
		"localhost",
		"127.0.0.1"
```

To cover local envs too.

Comment (psmyrek) [reply docs/umberto.json:17]: Please add also `fake.ckeditor.com` to excluded domains.

**Review by psmyrek — APPROVED** [review]
LGTM 🎉

**Review by pomek — APPROVED** [review]
LGTM.

**Pull request merged**